lib/legacy/legacy-element-mixin.html

```diff
@@ -86,6 +86,17 @@
         this._applyListeners();
       }
 
+      /**
+       * Forwards `importMeta` from the prototype (i.e. from the info object
+       * passed to `Polymer({...})`) to the static API.
+       *
+       * @return {!Object} The `import.meta` object set on the prototype
+       * @suppress {missingProperties}
```

Comment:
Please add reasoning.

\`this\` is always in the instance in closure for some reason.